src/mono/mono/mini/interp/interp.c

```diff
@@ -406,7 +406,8 @@ get_context (void)
 		context->stack_start = (guchar*)mono_valloc (0, INTERP_STACK_SIZE, MONO_MMAP_READ | MONO_MMAP_WRITE, MONO_MEM_ACCOUNT_INTERP_STACK);
 		context->stack_end = context->stack_start + INTERP_STACK_SIZE - INTERP_REDZONE_SIZE;
 		context->stack_real_end = context->stack_start + INTERP_STACK_SIZE;
-		context->stack_pointer = context->stack_start;
+		/* We reserve an object pointer slot at the top of the interp stack to make temp objects visible to GC */
+		context->stack_pointer = context->stack_start + sizeof(MonoObject*);
```

Comment:
We always keep the stack pointer and all interp vars aligned to `MINT_STACK_SLOT_SIZE`, which is 8 bytes.  On 32 bit this breaks alignment. This doesn't appear to be a critical problem, but it would be best to maintain the default alignment.

Reply:
Will fix it, thanks.